### PR TITLE
[Hub Generated] Review request for Microsoft.HybridCompute to add version stable/2020-08-15

### PR DESCRIPTION
**Pull request by Atchub**
The PR adds PrivateLinkScopes.Json and examples, rest of the files are from previous version, Please dont merge the change as I need to change the base API version. Please review the changes as its a new resource addtion and is being targetted for Ignite. thanks!

This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/Atchub/azure-rest-api-specs/atchub-hybridcompute-Microsoft.HybridCompute-2020-08-15...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by Atchub** [comment]
@xccc-msft , can you please review the PR?

**Comment by xccc-msft** [comment]
@Atchub I'm from SDK team. `WaitForARMFeedback` requires ARM team to review for your new APIs.

**Review by ravbhatnagar — COMMENTED** [review]
Signing off from ARM side.

**Review by xseeseesee — APPROVED** [review]

**Comment by xccc-msft** [comment]
@Atchub I cannot find hybridCompute in [CODEOWNERS](https://github.com/Azure/azure-rest-api-specs/blob/master/CODEOWNERS). Do you know who can help review from service end?

File: specification/hybridcompute/resource-manager/readme.md [hunk 1]
```diff
@@ -85,6 +191,31 @@ See configuration in [readme.csharp.md](./readme.csharp.md)
...
+```
+
+### Tag: package-2019-12 and go
+
+These settings apply only when `--tag=package-2019-12 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2019-12' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/hybridcompute/mgmt/2019-12-12/hybridcompute
+```
```
Comment (TylerLeonhardt) [inline specification/hybridcompute/resource-manager/readme.md:217]: please put this in the readme.go.md

File: specification/hybridcompute/resource-manager/readme.md [hunk 2]
```diff
@@ -75,6 +88,99 @@ swagger-to-sdk:
...
+    - Machines_CreateOrUpdate
+    - Machines_Update
+    - MachineExtensions_CreateOrUpdate
+    - MachineExtensions_Update
+    - MachineExtensions_Delete
+    - MachineExtensions_Get
+    - MachineExtensions_List
+```
+
+## Python
```
Comment (TylerLeonhardt) [inline specification/hybridcompute/resource-manager/readme.md:135]: please put this in the readme.python.md

File: specification/hybridcompute/resource-manager/readme.md [hunk 3]
```diff
@@ -75,6 +88,99 @@ swagger-to-sdk:
...
+    - Machines_CreateOrUpdate
+    - Machines_Update
+    - MachineExtensions_CreateOrUpdate
+    - MachineExtensions_Update
+    - MachineExtensions_Delete
+    - MachineExtensions_Get
+    - MachineExtensions_List
+```
+
+## PowerShell
```
Comment (TylerLeonhardt) [inline specification/hybridcompute/resource-manager/readme.md:116]: please remove this section and do what was already there.

File: specification/hybridcompute/resource-manager/readme.md [hunk 4]
```diff
@@ -75,6 +88,99 @@ swagger-to-sdk:
 ```
 
 ---
+
+## C#
```
Comment (TylerLeonhardt) [inline specification/hybridcompute/resource-manager/readme.md:92]: please put this in the readme.csharp.md

**Review by TylerLeonhardt — CHANGES_REQUESTED** [review]
Please take a look at the README files in master

**Comment by xccc-msft** [comment]
Close this as changes merged in another PR. Please feel free to reopen if you still need to work on this one. Thanks.

**Pull request closed without merging**